src/main/java/mate/academy/intro/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.intro.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
let's make this exception extended from RuntimeException to reduce boilerplate code. 
```suggestion
public class RegistrationException extends RuntimeException {
```

Reply:
make sure you handle this exception using GlobalHandler